applications/embeddings-modal/src/embeddings.py

```diff
@@ -0,0 +1,55 @@
+from sentence_transformers import SentenceTransformer
+from functools import lru_cache
+from pydantic import BaseModel, Field
+from typing import List, Union
+
+model_names = {
```

Comment:
A bit unsure on how to integrate the various model versions when we're passing in the model name through the request body. I've currently got the model name set to what the user passes in, but I'm also considering an alternative where the request body will pass in like

`"type": "base"`, `"type": "small"`, or `"type": "large"`

rather than `"model": "BAAI/bge-base-en-v1.5"`...